src/Appwrite/SDK/Specification/Format/OpenAPI3.php

```diff
@@ -401,6 +401,7 @@ public function parse(): array
                         $node['schema']['x-example'] = '<' . \strtoupper(Template::fromCamelCaseToSnake($node['name'])) . '>';
                         break;
                     case 'Utopia\Validator\Boolean':
+                    case 'Appwrite\Utopia\Database\Validator\Spatial':
```

Comment:
If we set:

```
$node['schema']['items'] = [
    'type' => '{}',
];
```

We should hopefuly get "list of any", since swagger spec itself doesn't even support "list of list"